people-details-sql-jdbc/src/main/java/menu/Menu.java

```diff
@@ -0,0 +1,45 @@
+package menu;
+
+import dao.PersonDAO;
+import loader.PropertiesLoader;
+import org.jline.reader.LineReader;
+import org.jline.reader.LineReaderBuilder;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Menu {
+    public static void start() {
+        try {
+            LineReader lineReader = LineReaderBuilder.builder().build();
+            PropertiesLoader propertiesLoader = new PropertiesLoader();
+            Connection connection = DriverManager.getConnection(propertiesLoader.getUrl(), propertiesLoader.getUserName(), propertiesLoader.getPassword());
+            PersonDAO personDAO = new PersonDAO(connection, lineReader);
+            ListAllPerson listAllPerson = new ListAllPerson();
+            menu(lineReader, personDAO, listAllPerson);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static void menu(LineReader lineReader, PersonDAO personDAO, ListAllPerson listAllPerson) {
```

Comment:
again, a method should be a verb phrase in general, so i think we could find a better name.
another thing is to use the DAO interface as a param (programming to an interface), so if you want / need to switch to another implementation in this method, it can go pretty easily